src/course/syllabus/application/four/project.md

```diff
@@ -1 +1,19 @@
-project
+# Fruit Shop
+
+Your Project this week is to build a fruit shop app that lists produce and allows a user to create a shopping basket!
+
+{% box %}
+For this project, we also expect you to research how to use git from the command line, instead of using the source control tab on VS code.
+
+If you have not used the terminal/command line before - it's also a good idea to do some research on navigating your file system using the terminal, and using commands like `cd`, `ls`and `touch`/`echo`
+{% endbox %}
```

Comment:
Small markdown/shortcodes thing: if you want the content inside a shortcode to be treated as markdown it needs to have linebreaks around it. E.g.
```md
{% box %}
_hello_
{% endbox %}
```
will produce
```html
<div class="box">
_hello_
</div>
```
whereas
```md
{% box %}

_hello_

{% box %}
```
will produce
```html
<div class="box">
<p><em>hello</em></p>
</div>
```